### PR TITLE
fix: added favicon.ico

### PR DESCRIPTION
**Pull request by ayushpaharia**
Issue #18 

I was running the project by - 
`yarn install` to `yarn typedocGen` to `yarn typedocServe`

The server seems to run properly here 

Clicking on any subpage like
image

Throws error
```
Request for /favicon.ico by method GET
node:events:491
      throw er; // Unhandled 'error' event
      ^

Error: ENOENT: no such file or directory, open '/home/ayush/Desktop/box/solana/dappio/gateway/src/doc/favicon.ico'
Emitted 'error' event on ReadStream instance at:
    at emitErrorNT (node:internal/streams/destroy:157:8)
    at emitErrorCloseNT (node:internal/streams/destroy:122:3)
    at processTicksAndRejections (node:internal/process/task_queues:83:21) {
  errno: -2,
  code: 'ENOENT',
  syscall: 'open',
  path: '/home/ayush/Desktop/box/solana/dappio/gateway/src/doc/favicon.ico'
}
error Command failed with exit code 1.
```

I added a `favicon.ico` to `/src` that solves this issue.

### CODE REVIEW

**Comment by alvinphilips** [comment]
Hey @ayushpaharia it looks like your IDE also stripped the terminating `;`s from `typedocServe.js`. I think that should be rolled back :)

**Comment by alvinphilips** [comment]
@ironaddicteddog perhaps we should add prettier to this too?

**Comment by ironaddicteddog** [comment]
Yes, this needs to be prettified too

**Review by ayushpaharia — COMMENTED** [review]
reviewed changes and rebased to master to include changes from latest prettier PR

**Comment by alvinphilips** [comment]
Hey, it looks like you also committed your ids.ts file with the marinade program key

File: src/ids.ts [hunk 1]
```diff
@@ -59,3 +59,7 @@ export const FRANCIUM_ADAPTER_PROGRAM_ID = new PublicKey(
 export const FRIKTION_ADAPTER_PROGRAM_ID = new PublicKey(
   "ADPTzbsaBdXA3FqXoPHjaTjPfh9kadxxFKxonZihP1Ji"
 );
+
+export const MARINADE_ADAPTER_PROGRAM_ID = new PublicKey(
```
Comment (ironaddicteddog) [inline src/ids.ts]: Perhaps this should belong to another PR?

Comment (ayushpaharia) [reply src/ids.ts]: Yes makes sense

**Comment by ironaddicteddog** [comment]
Resolved by #23
